lms/services/blackboard_api.py

```diff
@@ -59,28 +46,8 @@ def get_token(self, authorization_code):
             validated_data.get("expires_in"),
         )
 
-    def list_files(self, course_id):
-        """Return the list of files in the given course."""
-        response = self._http_service.get(
-            self._api_url(f"courses/uuid:{course_id}/resources"), oauth=True
-        )
-
-        return BlackboardListFilesSchema(response).parse()
-
-    def public_url(self, course_id, document_url):
-        file_id = DOCUMENT_URL_REGEX.search(document_url)["file_id"]
-
-        try:
-            response = self._http_service.get(
-                self._api_url(f"courses/uuid:{course_id}/resources/{file_id}"),
-                oauth=True,
-            )
-        except HTTPError as err:
-            if err.response.status_code == 404:
-                raise BlackboardFileNotFoundInCourse(file_id) from err
-            raise
-
-        return BlackboardPublicURLSchema(response).parse()
+    def request(self, method, path):
+        return self._http_service.request(method, self._api_url(path), oauth=True)
```

Comment:
The `request()` method is very simple for now but more functionality will be added to it in future PRs